dbclient/dbclient.py

```diff
@@ -33,6 +33,11 @@ def __init__(self, configs):
         self._is_verbose = configs['verbose']
         self._verify_ssl = configs['verify_ssl']
         self._file_format = configs['file_format']
+
+        self._overwrite = 'false'
+        if('overwrite' in configs):
+            self._overwrite = str(configs['overwrite']).casefold()
```

Comment:
The config should store a boolean, so we should store a boolean here. There's no need to cast to string here.